### PR TITLE
Update OpenStack-Ansible SHA to tip of liberty

### PR DESCRIPTION
**Pull request by git-harry**
This commit updates the OpenStack-Ansible submodule to use the tip of
the liberty branch.

This update addresses one issue currently being tracked where the
Percona signing key for Apt packages has changed.

Connected https://github.com/rcbops/rpc-openstack/issues/1515

### CODE REVIEW

**Comment by odyssey4me** [comment]
👍 The SHA matches the current tip of Liberty

File: openstack-ansible [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 2821da8aa9fe006b7d151754af4c09152d97aff4
+Subproject commit 0193c0de2a736ab89d0dfe8d9791e216a4a42c92
```
Comment (hughsaunders) [inline openstack-ansible]: Correct SHA for current head of OSA liberty

**Comment by git-harry** [comment]
This is blocked on an upstream bug, @hughsaunders is in the process of submitting a fix.

**Comment by evrardjp** [comment]
I don't know how this SHA can be good, the proper commits have not yet merged...

**Comment by alextricity25** [comment]
Could someone link the upstream bugs, please?

**Comment by mattt416** [comment]
@alextricity25, guessing it's https://review.openstack.org/#/c/388685/ we're waiting on.

**Comment by git-harry** [comment]
The [additional upstream fix](https://review.openstack.org/#/c/388685/) is now included and this is ready for review.

**Comment by alextricity25** [comment]
:+1:

**Comment by prometheanfire** [comment]
:+1:

**Pull request merged**